section3/exercises/hashes.rb

```diff
@@ -1,31 +1,38 @@
 # In the below exercises, write code that achieves
 # the desired result. To check your work, run this
-# file by entering the following command in your terminal: 
+# file by entering the following command in your terminal:
 # `ruby section3/exercises/hashes.rb`
 
 # Example: Write code that prints a hash holding grocery store inventory:
 foods = {apples: 23, grapes: 507, eggs: 48}
 p foods
 
 # Write code that prints a hash holding zoo animal inventory:
-zoo = #YOUR CODE HERE
-p zoo
+zoo = {zebra: 3, sloths: 2, tigers: 4}
+puts "There are #{zoo[:zebra]} zebra, #{zoo[:sloths]} sloths, and #{zoo[:tigers]} tigers in the zoo."
 
-# Write code that prints all of the 'keys' of the zoo variable 
+
+# Write code that prints all of the 'keys' of the zoo variable
 # you created above:
-# YOUR CODE HERE
+puts zoo.keys
 
-# Write code that prints all of the 'values' of the zoo variable 
+# Write code that prints all of the 'values' of the zoo variable
 # you created above:
-# YOUR CODE HERE
+puts zoo.values
+
+# OR
 
-# Write code that prints the value of the first animal of the zoo variable 
+puts "#{zoo[:zebra]}, #{zoo[:sloths]}, #{zoo[:tigers]}"
+
+# Write code that prints the value of the first animal of the zoo variable
 # you created above:
 # YOUR CODE HERE
+puts zoo.fetch(:zebra)
```

Comment:
Very cool find

Challenge yourself: What if you didn't know the key for the first animal in the zoo variable? Could you call your first value in another way?